### DISCUSSION TITLE
[question][request feature] possible to feed UVR5 a specific sound to remove that sound?

### DISCUSSION THREAD

**Discussion by huangq12** [Anjok07/ultimatevocalremovergui#552] [Q&A]
possible to feed UVR5 a specific sound to remove that sound?
like remove whatsapp sound or sms notification sound

**Comment by borisbritvus** [comment]
Try Adobe Audition's Sound Remover function: https://creativecloud.adobe.com/learn/audition/web/audition-sound-remover-effect-cc?playlist=/services/playlist.helpx/products:SG_AUDITION_1_1/learn-path:key-techniques/set-header:effects/playlist:topic/en_us.json&ref=helpx.adobe.com

**Reply by luca-brwn** [reply]
i second this, absolutely great. another option is to have the source sound and invert it so it gets cancelled out, but that can be a hassle.

**Reply by borisbritvus** [reply]
Oh, I tried that as well, but that needs the sound to be the exact loudness and spectrogram... So it works almost never, and I not consider that as method at all :-)
